src/components/ShareBlock/ShareBlock.tsx

```diff
@@ -28,28 +27,27 @@ import {
   SocialTextAreaWrapper,
   EmbedPrompt,
 } from './ShareBlock.style';
-import { STATES } from 'common';
 import { trackShare } from 'components/Analytics';
+import { Region } from 'common/regions';
+import { STATES } from 'common';
```

Comment:
Consider using `regions.states.map(state => state.stateCode)` instead of `STATES`

Reply:
💯

Reply:
Oh this is actually just for the location page match, used as a type for the keys